@xen-orchestra/lite/src/components/AppHeader.vue

```diff
@@ -4,20 +4,34 @@
       <img alt="XO Lite" src="../assets/logo.svg" />
     </RouterLink>
     <slot />
-    <div>
-      <span @click="toggleTheme"
-        ><FontAwesomeIcon style="font-size: 1.5em" :icon="colorModeIcon"
-      /></span>
-      <span style="margin-left: 1rem" @click="logout">Logout</span>
+    <div style="display: flex; align-items: center; gap: 1rem">
+      <FontAwesomeIcon
+        :icon="colorModeIcon"
+        style="font-size: 1.5em"
+        @click="toggleTheme"
+      />
+      <span @click="logout">Logout</span>
+      <FormWidget :before="faEarthAmericas">
+        <select v-model="$i18n.locale">
+          <option v-for="locale in $i18n.availableLocales" :key="locale">
+            {{ locale }}
+          </option>
+        </select>
```

Comment:
The `locale` value should be stored in the `localStorage` to be used as the default language when launching `xo-lite`

Reply:
Right. As I said, at some point I have to jump in and do a PR. This feature will be added next :)

Reply:
Right. As I said, at some point I have to jump in and do a PR. This feature will be added next, along with the detection of the browser language during the first initialization :)